lib/vrt/cross_version_mapping.rb

```diff
@@ -5,7 +5,7 @@ module CrossVersionMapping
     def cross_version_category_mapping
       category_map = {}
       deprecated_node_json.each do |key, value|
-        latest_version = value.keys.sort_by { |n| Gem::Version.new(n) }.last
+        latest_version = value.keys.max_by { |n| Gem::Version.new(n) }
```

Comment:
Let's update this so that it will ping @maschwenk on Slack whenever it's used.  :trollface:

Reply:
Lol curse you!